src/components/Articles.jsx

```diff
@@ -1,10 +1,13 @@
-import { useState } from 'react';
 import ArticleCard from './ArticleCard';
 import { Typography, Space } from 'antd';
+import { Bars } from 'react-loading-icons';
 
-const Articles = ({ articles }) => {
+const Articles = ({ articles, isLoading }) => {
 	const { Title } = Typography;
-	const [isLoading, setIsLoading] = useState(false);
```

Comment:
Can see that you've removed the `isLoading` state from here up into the App - I would revert that decision David!
You don't really want a global loading state otherwise your app is waiting for EVERYTHING to load before you show anything to the user.
It also looks like you'd be able to move the articles state down into this component